templates/zerver/emails/missed_message.source.html

```diff
@@ -27,6 +27,8 @@
     &mdash;<br>
     {% if mention %}
     You are receiving this because you were mentioned in {{ realm_name }}.<br>
+    {% elif topic_email_notify %}
+    You are receiving this because you have email notifications enabled for topics you've contributed to.<br>
```

Comment:
I think "because you contributed to this topic" would be better for brevity?

Reply:
Also @hackerkid do we still not have i18n for this section of our emails configured?